.changeset/chubby-geese-arrive.md

```diff
@@ -0,0 +1,5 @@
+---
+'@clerk/backend': patch
+---
+
+Renaming DEFAULT_CLOCK_SKEW_IN_SECONDS to DEFAULT_CLOCK_SKEW_IN_MS which is the actual value
```

Comment:
If we want, as this is just an internal change, we can provide an empty changeset:
```suggestion
---
---
```

This can also be done with `px changeset --empty`!